source/release-notes.html.md

```diff
@@ -36,6 +36,8 @@ Additional changes:
 - Add description of the API versioning
 - Add error responses to OpenAPI spec for application endpoints
 - Add instructions for importing our OpenAPI specification in the Swagger Editor
+- Document the workflow for authenticating with an API key
```

Comment:
Maybe we can add it to the spec as well?

https://swagger.io/docs/specification/authentication/bearer-authentication

It won't be visible in our docs, but will in Swagger Editor I think.

```
components:
  securitySchemes:
    bearerAuth:            # arbitrary name for the security scheme
      type: http
      scheme: bearer
      bearerFormat: JWT    # optional, arbitrary value for documentation purposes
  - bearerAuth: []         # use the same name as above
```

Reply:
Added 🔒